test/integration/index.spec.js

```diff
@@ -356,6 +354,33 @@ describe('/tables', async () => {
     await axios.delete(`${URL}/columns/${newTable.id}.1`)
     await axios.delete(`${URL}/tables/${newTable.id}`)
   })
+  it('POST /columns with unique constraint', async () => {
+    const { data: newTable } = await axios.post(`${URL}/tables`, { name: 'foo' })
+    await axios.post(`${URL}/columns`, {
+      table_id: newTable.id,
+      name: 'bar',
+      type: 'int2',
+      is_unique: true,
+    })
+
+    const { data: uniqueColumns } = await axios.post(
+      `${URL}/query`,
+      { query: `
+        SELECT a.attname
+        FROM   pg_index i
+        JOIN   pg_constraint c ON c.conindid = i.indexrelid
```

Comment:
Technically speaking looking at the index alone should be enough (PG uses a [unique index](https://postgresql.org/docs/13/interactive/indexes-unique.html) to enforce a unique constraint) but I added this for completeness, since we specifically want to create a constraint.